setup.cfg

```diff
@@ -33,7 +33,7 @@ install_requires =
     django-saml2-auth==2.2.1
     djangorestframework==3.12.2
     drf-spectacular==0.15.1
-    tycho-api==1.14.1
+    tycho-api==1.15.0
```

Comment:
Still cool with removing this entirely if you'd like.

Reply:
I don't think I'll miss it.